packages/cdk/src/code-build-project.ts

```diff
@@ -58,7 +58,9 @@ export class CodeBuildProject extends Construct {
       source: gitHubSource,
       environment: {
         buildImage:
-          props.nodeVersion === '18' ? codebuild.LinuxBuildImage.STANDARD_7_0 : codebuild.LinuxBuildImage.STANDARD_6_0,
+          props.nodeVersion === '18' || props.nodeVersion === '20'
+            ? codebuild.LinuxBuildImage.STANDARD_7_0
+            : codebuild.LinuxBuildImage.STANDARD_6_0,
```

Comment:
Node 16 ceases to be supported by the SDK in January 2025. Perhaps flip the test ? Standard 6.0 for Node 16, otherwise go ahead with Standard 7 ?

Reply:
if I remember correctly, there were issues last time when we switched the default, as some projects are still "watched" by pipeline factory and automatically updated or something